backend/src/main/scala/cromwell/backend/standard/callcaching/StandardFileHashingActor.scala

```diff
@@ -91,7 +93,7 @@ abstract class StandardFileHashingActor(standardParams: StandardFileHashingActor
     }
 
     ioHashCommandTry match {
-      case Success(ioHashCommand) => sendIoCommandWithContext(ioHashCommand, fileRequest)
+      case Success(ioHashCommand) => sendIoCommandWithContext(ioHashCommand, FileHashContext(fileRequest.hashKey, fileRequest.file.value))
```

Comment:
This is unrelated, the reason is while trying to see if this fixed the problem I noticed Cromwell was spending a ridiculous amount of time trying to compute the hashcode of the `fileRequest` as it's being inserted into an hashmap. This reduces the hashing to only what's needed.